packages/core/src/textures/TextureSystem.ts

```diff
@@ -342,6 +342,11 @@ export class TextureSystem implements ISystem
                 glTexture.internalFormat = gl.RGB16F;
             }
         }
+
+        if (this.webGLVersion === 2 && texture.internalFormat)
```

Comment:
just curious, but do internal formats only work with WebGL2? or are there just more available?

As this function is getting a bit larger perhaps we tweak to:

```
glTexture.internalFormat = texture.internalFormat || autodetectInternalFormat(texture) ;
```